tmt/utils.py

```diff
@@ -554,11 +554,15 @@ def _run(self,
             environment = None
         self.debug('environment', pprint.pformat(environment), level=4)
 
+        # Set only for shell=True as it would affect command
+        executable = "/bin/bash" if shell else None
```

Comment:
I'd suggest moving `/bin/bash` to a module-level "constant", `DEFAULT_SHELL_EXECUTABLE` maybe? "No magic numbers in code" should apply to "magic" strings as well.

Reply:
Sounds reasonable, will update that.